src/common/drydep.f90

```diff
@@ -103,4 +162,255 @@ subroutine drydep2(tstep, part)
     depdry(i,j,mm) = depdry(i,j,mm) + dble(dep)
   end if
 end subroutine drydep2
+
+pure function aerodynres(L, ustar, z0) result(raero)
+  real(real64), intent(in) :: L, ustar, z0
+  real(real64) :: raero
+
+  real(real64), parameter :: ka = 0.4
+  real(real64), parameter :: z = 30 ! Assumed height of surface/constant flux layer
```

Comment:
is z in line 171 == h in line 138? Should be one parameter for the complete ddep-code.